### PR TITLE
derive_more

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
r? @Emilgardis 
I removed `prebuild` task in favor of independence caches as sometimes it does not work correctly.

Saying about main change, it adds optional deriving of `Deref` and `From` traits which significantly decrease boilerplate, but require advanced dependency and can slow down compilation, especially first one.

File: src/util.rs [hunk 1]
```diff
@@ -45,6 +46,7 @@ impl Default for Config {
             keep_list: false,
             strict: false,
             pascal_enum_values: false,
+            derive_more: false,
```
Comment (Emilgardis) [inline src/util.rs:49]: I think the default should be that it's enabled. That probably means the flag needs to be negated, e.g `derive_less` perhaps

**Review by Emilgardis — CHANGES_REQUESTED** [review]
over all looks good, just one concern with defaults

File: src/util.rs (hunk 1 above)
Comment (Emilgardis) [reply src/util.rs:49]: `no_derive_more` ?

Comment (Emilgardis) [reply src/util.rs:49]: I take that back, it should not be defaulted to on

**Review by Emilgardis — DISMISSED** [review]
im fine with this, no concerns

File: src/util.rs (hunk 1 above)
Comment (burrbull) [reply src/util.rs:49]: As this requires advanced dependency, it is breaking change for all svd2rust-generated crates.
So I don't think it is good idea to make it default.

**Comment by burrbull** [comment]
conflict. again. fixed

**Comment by burrbull** [comment]
cc @Emilgardis

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**